build.gradle

```diff
@@ -26,6 +27,11 @@ configurations {
 intellij {
   version 'LATEST-EAP-SNAPSHOT'
   alternativeIdePath  project.hasProperty("idePath") ? project.findProperty("idePath") : ""
+
+// Convenience tool for development to include other plugins
+//  setPlugins(
+//    'indent-rainbow.indent-rainbow:1.5.1'
+//  )
```

Comment:
May want to throw this in a conditional block to safeguard against uncommenting it and forgetting to re-comment before releasing.

Reply:
Good Idea!

Reply:
Done!